### PR TITLE
Add an example for DeepFM estimator in deepctr

### PR DESCRIPTION
**Pull request by workingloong**

### CODE REVIEW

File: model_zoo/deepctr/deepfm_estimator.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+def arg_parser():
+    parser = argparse.ArgumentParser(description="Process training parameters")
+    parser.add_argument("--training_data", type=str, required=True)
+    parser.add_argument(
+        "--validation_data", type=str, default="", required=False
+    )
+    return parser
+
+
+class ElasticDataShardReportHook(tf.train.SessionRunHook):
```
Comment (brightcoder01) [inline model_zoo/deepctr/deepfm_estimator.py]: We can move this common hook into elasticai_api

Comment (workingloong) [reply model_zoo/deepctr/deepfm_estimator.py]: Good idea！

File: docs/tutorials/elasticdl_deepctr_estimator.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+# Distributed Training of DeepCTR Estimator using ElasticDL on Kubernetes
+
+This document shows how to run an distributed training job of a deepctr
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr_estimator.md]: an -> a

File: docs/tutorials/elasticdl_deepctr_estimator.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  --evaluator_resource_limit="cpu=1,memory=2048Mi" \
+  --job_name=test-deepfm-estimator \
+  --distribution_strategy=ParameterServerStrategy \
+  --need_tf_config=true \
+  --volume="host_path={criteo_data_path},mount_path=/data" \
+```
+
+`--image_name` is the image to launch the ElasticDL master which
+has nothing to do with the estimator model. The ElasticDL master is
+responsible for launching pod and assign data shard to workers with
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr_estimator.md]: assign => assigning
data shard => data shards

File: docs/tutorials/elasticdl_deepctr_estimator.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+elasticity and fault-tolerance.
+
+`{criteo_data_path}` is the absolute path of the `./data` with `criteo_sample.txt`.
+Here, the option `--volume="host_path={criteo_data_path},mount_path=/data"`
+bind mount it into the containers/pods.
+
+The option `--num_workers=1` tells the master container to start a worker pod.
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr_estimator.md]: the master container -> the master

File: docs/tutorials/elasticdl_deepctr_estimator.md (hunk 2 above)
Comment (workingloong) [reply docs/tutorials/elasticdl_deepctr_estimator.md]: Done!

File: docs/tutorials/elasticdl_deepctr_estimator.md (hunk 3 above)
Comment (workingloong) [reply docs/tutorials/elasticdl_deepctr_estimator.md]: Done

**Review by brightcoder01 — APPROVED** [review]

**Pull request merged**